aws/app/lambda/submission/submission.js

```diff
@@ -65,6 +66,7 @@ const saveData = async (submissionID, formData) => {
       SendReceipt: { S: "unknown" },
       FormSubmissionLanguage: {S: formData.language},
       FormData: { S: formSubmission },
+      SubmissionDateTime: {S: (new Date()).toISOString()}
```

Comment:
I think we should generate a timestamp and save it as a number in Dynamo. User facing format should come at the end when we need to display the date for the user. It is also relevant to have a number if we create an index afterwards. Especially for the next feature that is coming https://github.com/cds-snc/platform-forms-client/issues/242 which would benefit from a timestamp.

Reply:
The first implementation was `new Date().getTime()` which already covered this scenario.  I kinda agree with you in regard to the indexe.  @Moro-Code suggested  the use of ISO format instead.

Reply:
@Moro-Code I am really thinking in terms of value it would bring to the GCNotify callback feature I am working on. But also I feel like it would be more flexible in the future if we save a timestamp rather than a fully formatted date string which can be derived from the timestamp

Reply:
@Moro-Code @craigzour if there's no more comments on this matter, I'll assume it's safer to go with the number of milliseconds a.k.a `getTime`.  TY

Reply:
You make a good case @craigzour and @falila a number works